models/rotors_description/urdf/vtol_base.xacro

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0"?>
+<!--
+  Copyright 2015 Fadri Furrer, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Michael Burri, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Mina Kamel, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Janosch Nikolic, ASL, ETH Zurich, Switzerland
+  Copyright 2015 Markus Achtelik, ASL, ETH Zurich, Switzerland
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <!-- Main multirotor link -->
+  <xacro:macro name="vtol_base_macro" params="robot_namespace *origin">
+    <xacro:insert_block name="origin" />
+    <link name="base_link"></link>
+    <joint name="base_joint" type="fixed">
+      <parent link="base_link" />
+      <child link="base_link_inertial" />
+    </joint>
+    <link name="base_link_inertial">
+      <inertial>
+        <mass value="1.5" />
+        <inertia ixx="0.477708333333" ixy="0.0" iyy="0.341666666667" ixz="0.0" iyz="0.0" izz="0.811041666667" />
+      </inertial>
+      <visual>
+        <origin xyz="0.53 -1.072 -0.1" rpy="1.5707963268 0 3.1415926536" />
+        <geometry>
+            <mesh scale="0.001 0.001 0.001" filename="model://standard_vtol/meshes/x8_wing.dae" />
+        </geometry>
+      </visual>
+      <!-- <visual name='left_motor_column'>
```

Comment:
Do we really need all this commented code here?

Reply:
No, it was commented out from the original sdf file since xacro doesn't seem to be able to handle multiple visual elements within a link. Removed now!